### DISCUSSION TITLE
Having trouble theming some components

### DISCUSSION THREAD

**Discussion by psiho** [segmentio/evergreen#1244] [Q&A] [answered]
I'm trying to change appearance of disabled Select dropdown. Wanted to completely remove borders, set transparent background, set text cursor, and remove icon so it all appears as a pure text. So what I did was:

```
defaultTheme.components.Select.baseStyle.disabled = {
    	backgroundColor: 'transparent',
	border: 'none',
	paddingLeft: 0,
	color: 'black',
	cursor: 'text'
}
```

... but  color and cursor get overwritten and WebkitAppearance for the arrow is already set to 'none', I guess the icon is done some in other way.

How would you solve this?

**Comment by akleiner2** [comment, marked as answer]
@psiho Instead of adding these styles in `baseStyle` under the `disabled` key, you might want to try something like this: 


```
defaultTheme.component.Select.appearances.default = { 
    ...defaultTheme.component.Select.appearances.default, 
   _disabled: {
     // your styles here
    }
}

```
